src/utilities/auth/loginHandler.js

```diff
@@ -68,6 +70,15 @@ const loginHandler = async (
           playlists: responsePlaylists.data.playlists,
         },
       });
+      const responseWatchlater = await getWatchlaterService(
+        response.data.encodedToken
+      );
+      watchlaterDispatch({
+        type: "INITIAL_WATCHLATER",
+        payload: {
+          watchlater: responseWatchlater.data.watchlater,
+        },
+      });
       componentsDispatch({
```

Comment:
Check here for response. status and then send a dispatch.

Reply:
Thanks for pointing it out! I might have missed that. I'll correct that.